hypershift-operator/controllers/hostedcluster/hostedcluster_controller.go

```diff
@@ -2000,9 +2013,15 @@ func reconcileCAPIAWSProviderDeployment(deployment *appsv1.Deployment, hc *hyper
 		},
 	}
 	hyperutil.SetColocation(hc, deployment)
-	hyperutil.SetRestartAnnotation(hc, deployment)
```

Comment:
I'm dropping this since I'm not sure there's a known reason to have it in a component different than the genuine control plane components. If we happen to see scalability issues not trivially solvable we can add it then.

Reply:
same comment of can we just comment it out and put a TODO to revealutate? I ultimately think it will still be necessary but am ok with enabling at a later time. (just want the todo to be able to easily turn on)